Userland/Libraries/LibCore/Stream.h

```diff
@@ -736,8 +736,16 @@ class BufferedSeekable final : public SeekableStream {
     BufferedHelper<T> m_helper;
 };
 
+class BufferedSocketBase : public Socket {
+public:
+    virtual ErrorOr<size_t> read_line(Bytes buffer) = 0;
+    virtual ErrorOr<size_t> read_until(Bytes buffer, StringView const& candidate) = 0;
```

Comment:
Please pass `StringView` by value.

Reply:
This isn't adding anything that wasn't there, but I agree, I'll make it take that by value.